subprojects/core/src/main/java/org/gradle/api/internal/project/DefaultProject.java

```diff
@@ -526,6 +526,14 @@ public Path getIdentityPath() {
     }
 
     @Override
+    public String getBuildOperationDisplayName() {
```

Comment:
We shouldn't need this new interface. Instead, we can just create subclasses of `BuildOperationAction` when we need to reuse some display name calculation logic. E.g. `ConfigureProjectAction`.